backend/src/test/groovy/pt/ulisboa/tecnico/socialsoftware/tutor/question/service/CreateQuestionTest.groovy

```diff
@@ -244,6 +245,158 @@ class CreateQuestionTest extends SpockTest {
         exception.getErrorMessage() == ErrorMessage.NO_CORRECT_OPTION
     }
 
+
```

Comment:
There is several code that is repeated between tests - create auxiliary methods to be reused in the tests